sdk/communication/azure-communication-identity/azure/communication/identity/__init__.py

```diff
@@ -34,18 +36,21 @@
     "identifier_from_raw_id",
     "MicrosoftTeamsUserIdentifier",
     "MicrosoftTeamsUserProperties",
+    "MicrosoftTeamsAppIdentifier",
+    "MicrosoftTeamsAppProperties",
     "PhoneNumberIdentifier",
     "PhoneNumberProperties",
     "UnknownIdentifier",
 ]
 
 def __getattr__(name):
     if name == 'MicrosoftBotProperties':
-        warnings.warn(f"{name} is deprecated and should not be used.", DeprecationWarning)
+        warnings.warn(f"{name} is deprecated and should not be used", DeprecationWarning)
         from ._shared.models  import _MicrosoftBotProperties
         return _MicrosoftBotProperties
```

Comment:
we will remove _MicrosoftBotProperties, instead return MicrosoftTeamsAppProperties

Reply:
Changed.